src/validators/address.validator.ts

```diff
@@ -27,3 +27,16 @@ export class AddressValidator implements Validator<string> {
         }
     }
 }
+
+export class AddressAliasValidator implements Validator<string> {
+    validate(value: string, context: ValidationContext): void {
+        const aliasTag = '@';
+        if (value.charAt(0) !== aliasTag) {
+            try {
+                Address.createFromRawAddress(value);
+            } catch (e) {
+                throw new ExpectedError('Introduce a valid address. Example: SBI774-YMFDZI-FPEPC5-4EKRC2-5DKDZJ-H2QVRW-4HBP');
+            }
+        }
```

Comment:
We could try to check if the namespace is valid. 

Pseudocode

```
else {
   const alias = value (wihtout @);
   try new NamespaceId(alias) catch throw new Error
}
```

Reply:
fixed